detect_secrets/plugins/high_entropy_strings.py

```diff
@@ -22,6 +32,14 @@ def __init__(self, charset, limit, *args):
         # TODO: Update for not just python comments?
         self.ignore_regex = re.compile(r'# ?pragma: ?whitelist[ -]secret')
 
+    def analyze(self, file, filename):
+        # Heuristically determine whether file is an ini-formatted file.
+        for ext in INI_FILE_EXTENSIONS:
```

Comment:
We could maybe run `file` on them and see if it comes back `ASCII text`, might get expensive.

Reply:
I'm kind of ambivalent, right now it's easier to exclude files than include them in the tuple, which makes me lean towards including more by default/making the false-p/false-n rate customizable. Nothing is really set in stone though, we can always change it in the future.

Reply:
Good point. Let me jiggle it up a bit.